internal/datastore/mysql/migrations/driver.go

```diff
@@ -98,16 +100,52 @@ func (driver *MySQLDriver) Version(ctx context.Context) (string, error) {
 	return "", errors.New("no migration version detected")
 }
 
+func (driver *MySQLDriver) Conn() Wrapper {
+	return Wrapper{db: driver.db, tables: driver.tables}
+}
+
+func (driver *MySQLDriver) RunTx(ctx context.Context, f migrate.TxMigrationFunc[TxWrapper]) error {
+	return BeginTxFunc(
+		ctx,
+		driver.db,
+		&sql.TxOptions{Isolation: sql.LevelSerializable},
```

Comment:
Just calling out that this was _recently added_ in this PR. When `nil` is provided, the default isolation level is used.